.github/workflows/docs.yml

```diff
@@ -26,7 +26,7 @@ jobs:
         run: |
           tox -edocs
       - name: Deploy docs
-        if: startsWith(github.ref, 'refs/heads/stable/') || github.ref == 'refs/heads/main'
+        #if: startsWith(github.ref, 'refs/heads/stable/') || github.ref == 'refs/heads/main'
```

Comment:
Note to myself : Do not forget to reverse

Reply:
@mickahell did you forget about this? :)

Reply:
Let's reverse it now, in case I need it after your review and the review of Eric I'll reapply it